src/js/components/Video/Video.js

```diff
@@ -254,10 +284,68 @@ const Video = forwardRef(
         onClick: () => showCaptions(caption.active ? -1 : 0),
       }));
 
+      const buttonProps = {
+        closedCaption: captionControls,
+        fullScreen: {
+          icon: <Icons.FullScreen color={iconColor} />,
+          a11yTitle: format({
+            id: 'video.fullScreen',
+            messages,
+          }),
+          onClick: fullscreen,
+        },
+        pause: {
+          icon: <Icons.Pause color={iconColor} />,
+          a11yTitle: format({
+            id: 'video.pauseButton',
+            messages,
+          }),
+          onClick: playing ? pause : play,
+        },
+        play: {
+          icon: <Icons.Play color={iconColor} />,
+          a11yTitle: format({
+            id: 'video.playButton',
+            messages,
+          }),
+          onClick: playing ? pause : play,
+        },
+        reduceVolume: {
+          icon: <Icons.ReduceVolume color={iconColor} />,
+          a11yTitle: format({
+            id: 'video.volumeDown',
+            messages,
+          }),
+          onClick: volume >= VOLUME_STEP ? quieter : undefined,
+          close: false,
+        },
+        volume: {
+          icon: <Icons.Volume color={iconColor} />,
+          a11yTitle: format({ id: 'video.volumeUp', messages }),
+          onClick: volume <= 1 - VOLUME_STEP ? louder : undefined,
+          close: false,
+        },
+      };
+
+      const controlsMenuItems =
+        controls.items &&
+        controls.items.map((item) => {
+          if (typeof item === 'string') {
+            return buttonProps[item];
+          }
+          return {
```

Comment:
How about just returning `item`? This will allow the caller to set other properties on the Button as needed. I think this is what Menu does.